tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetWebApi.cs

```diff
@@ -79,6 +80,8 @@ public AspNetWebApi(IisFixture iisFixture, ITestOutputHelper output, bool classi
         [InlineData(AddressesConstants.RequestQuery, "/api/Health/?arg=[$slice]", null)]
         [InlineData(AddressesConstants.RequestQuery, "/api/Health/?arg&[$slice]", null)]
         [InlineData(AddressesConstants.RequestPathParams, "/api/Health/appscan_fingerprint", null)]
+        [InlineData(AddressesConstants.RequestPathParams, "/api/route/2", null)]
+        [InlineData(AddressesConstants.RequestPathParams, "/api/route/TwoMember", null)]
```

Comment:
both these routes would cause the exception seen in the logs, and mean that the WAF wouldn't run without the proposed change.

Reply:
Update the routes to have an attack, so verify files show that the WAF runs.